examples/using-wordpress/gatsby-node.js

```diff
@@ -93,7 +93,7 @@ exports.createPages = ({ graphql, boundActionCreators }) => {
           // The Post ID is prefixed with 'POST_'
           _.each(result.data.allWordpressPost.edges, edge => {
             createPage({
-              path: edge.node.slug,
+              path: `/${edge.node.slug}/`,
```

Comment:
This change breaks prefetching, please revert this and leave just routes comment change (which is valid)

Reply:
OK. I fixed.